apps/aehttp/test/aehttp_contracts_SUITE.erl

```diff
@@ -1181,7 +1181,7 @@ remote_gas_test_contract(Config) ->
     [] = call_func(APub, APriv, EncC1Pub, Contract, "call", [ZeroContract, "2", "1"], error),
     force_fun_calls(Node),
     Balance5 = get_balance(APub),
-    ?assertMatchVM(900000, 800017, (Balance4 - Balance5) div ?DEFAULT_GAS_PRICE),
+    ?assertMatchVM(900000, 800141, (Balance4 - Balance5) div ?DEFAULT_GAS_PRICE),
```

Comment:
So I see the FATE execution became 8 times as expensive...

Reply:
Yes. Base cost is now 10/instruction instead of 3. But we will reduce the base tx.fee. And it will still be much cheaper than aevm.